Source/WebKit/UIProcess/DigitalCredentials/WKDigitalCredentialsPicker.h

```diff
@@ -52,16 +55,14 @@ struct OpenID4VPRequest;
 - (void)digitalCredentialsPickerDidDismiss:(WKDigitalCredentialsPicker *)picker;
 @end
 
-@interface WKDigitalCredentialsPicker : NSObject <WKDigitalCredentialsPickerDelegate>
+@interface WKDigitalCredentialsPicker : NSObject <WKDigitalCredentialsPickerDelegate, WKIdentityDocumentPresentmentDelegate>
 
 @property (nonatomic, weak) id<WKDigitalCredentialsPickerDelegate> delegate;
 
-
 - (instancetype)initWithView:(WKWebView *)view page:(WebKit::WebPageProxy*)page;
 
-- (void)presentWithRequestData:(const WebCore::DigitalCredentialsRequestData&)requestData completionHandler:(CompletionHandler<void(Expected<WebCore::DigitalCredentialsResponseData, WebCore::ExceptionData>&&)>&&)completionHandler;
-- (void)dismissWithCompletionHandler:(CompletionHandler<void(bool)>&&)completionHandler;
-
+- (void)presentWithRequestData:(const WebCore::DigitalCredentialsRequestData&)requestData completionHandler:(WTF::CompletionHandler<void(Expected<WebCore::DigitalCredentialsResponseData, WebCore::ExceptionData>&&)>&&)completionHandler;
```

Comment:
We want to avoid using the `WTF` namespace prefix in our project. We have `using WTF::CompletionHandler;` in `wtf/Forward.h`, so feel free to drop:
```suggestion
- (void)presentWithRequestData:(const WebCore::DigitalCredentialsRequestData&)requestData completionHandler:(CompletionHandler<void(Expected<WebCore::DigitalCredentialsResponseData, WebCore::ExceptionData>&&)>&&)completionHandler;
```